### PR TITLE
Fix RTL in refreshed Stats

### LINKED ISSUE #8581: Accessibility and RTL in new Stats
Recheck issues in #7287 to make sure we haven't broken accessibility in the Stats Refresh project. 
Check RTL behaviour in the new Stats to make sure it's not broken.

### PR DESCRIPTION
**Pull request by planarvoid**
Fixes #8581 
Fixes RTL issues in the new Stats after the refresh
* Fix of the Empty block - No data yet was aligned to the left
* Fix of the Labels - they were stuck together in the middle of the block
* Fix of the Link - the link was aligned to the left

To test:
* Set "Force RTL" in dev's settings
* Open Stats
* Check that everything is aligned to the right correctly

### CODE REVIEW

**Comment by malinajirka** [comment]
Thanks @planarvoid! I've left a couple of comments.

1. I think the rows in sections/blocks should be aligned right, wdyt?
screenshot_1547719883
screenshot_1547720047



2. I'm not sure it's related to these changes, but the app crashed when I swiped from Insights to Days

```
2019-01-17 10:24:03.770 9049-9049/org.wordpress.android.beta E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.wordpress.android.beta, PID: 9049
    java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:448)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
     Caused by: java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807) 
     Caused by: java.text.ParseException: Unparseable date: ""
        at java.text.DateFormat.parse(DateFormat.java:362)
        at org.wordpress.android.ui.stats.refresh.utils.StatsDateFormatter.parseStatsDate(StatsDateFormatter.kt:117)
        at org.wordpress.android.ui.stats.refresh.lists.sections.granular.usecases.OverviewUseCase.buildStatefulUiModel(OverviewUseCase.kt:84)
        at org.wordpress.android.ui.stats.refresh.lists.sections.granular.usecases.OverviewUseCase.buildStatefulUiModel(OverviewUseCase.kt:27)
        at org.wordpress.android.ui.stats.refresh.lists.sections.BaseStatsUseCase$StatefulUseCase.buildUiModel(BaseStatsUseCase.kt:165)
        at org.wordpress.android.ui.stats.refresh.lists.sections.BaseStatsUseCase$liveData$1.invoke(BaseStatsUseCase.kt:33)
        at org.wordpress.android.ui.stats.refresh.lists.sections.BaseStatsUseCase$liveData$1.invoke(BaseStatsUseCase.kt:22)
        at org.wordpress.android.util.LiveDataUtilsKt$merge$3.invoke(LiveDataUtils.kt:76)
        at org.wordpress.android.util.LiveDataUtilsKt$merge$3.invoke(Unknown Source:2)
        at org.wordpress.android.util.LiveDataUtilsKt$map$1.apply(LiveDataUtils.kt:132)
        at android.arch.lifecycle.Transformations$1.onChanged(Transformations.java:70)
        at android.arch.lifecycle.MediatorLiveData$Source.onChanged(MediatorLiveData.java:152)
        at android.arch.lifecycle.LiveData.considerNotify(LiveData.java:109)
        at android.arch.lifecycle.LiveData.dispatchingValue(LiveData.java:126)
        at android.arch.lifecycle.LiveData.setValue(LiveData.java:282)
        at android.arch.lifecycle.MutableLiveData.setValue(MutableLiveData.java:33)
        at org.wordpress.android.util.LiveDataUtilsKt$merge$1.onChanged(LiveDataUtils.kt:71)
        at android.arch.lifecycle.MediatorLiveData$Source.onChanged(MediatorLiveData.java:152)
        at android.arch.lifecycle.LiveData.considerNotify(LiveData.java:109)
        at android.arch.lifecycle.LiveData.dispatchingValue(LiveData.java:121)
        at android.arch.lifecycle.LiveData.access$400(LiveData.java:59)
        at android.arch.lifecycle.LiveData$ObserverWrapper.activeStateChanged(LiveData.java:416)
        at android.arch.lifecycle.LiveData.observeForever(LiveData.java:207)
        at android.arch.lifecycle.MediatorLiveData$Source.plug(MediatorLiveData.java:141)
        at android.arch.lifecycle.MediatorLiveData.onActive(MediatorLiveData.java:118)
        at android.arch.lifecycle.LiveData$ObserverWrapper.activeStateChanged(LiveData.java:410)
        at android.arch.lifecycle.LiveData.observeForever(LiveData.java:207)
        at android.arch.lifecycle.MediatorLiveData$Source.plug(MediatorLiveData.java:141)
        at android.arch.lifecycle.MediatorLiveData.onActive(MediatorLiveData.java:118)
        at android.arch.lifecycle.LiveData$ObserverWrapper.activeStateChanged(LiveData.java:410)
        at android.arch.lifecycle.LiveData.observeForever(LiveData.java:207)
        at android.arch.lifecycle.MediatorLiveData$Source.plug(MediatorLiveData.java:141)
        at android.arch.lifecycle.MediatorLiveData.onActive(MediatorLiveData.java:118)
        at android.arch.lifecycle.LiveData$ObserverWrapper.activeStateChanged(LiveData.java:410)
        at android.arch.lifecycle.LiveData.observeForever(LiveData.java:207)
        at android.arch.lifecycle.MediatorLiveData$Source.plug(MediatorLiveData.java:141)
        at android.arch.lifecycle.MediatorLiveData.onActive(MediatorLiveData.java:118)
        at android.arch.lifecycle.LiveData$ObserverWrapper.activeStateChanged(LiveData.java:410)
        at android.arch.lifecycle.LiveData.observeForever(LiveData.java:207)
        at android.arch.lifecycle.MediatorLiveData$Source.plug(MediatorLiveData.java:141)
2019-01-17 10:24:03.774 9049-9049/org.wordpress.android.beta E/AndroidRuntime:     at android.arch.lifecycle.MediatorLiveData.onActive(MediatorLiveData.java:118)
        at android.arch.lifecycle.LiveData$ObserverWrapper.activeStateChanged(LiveData.java:410)
        at android.arch.lifecycle.LiveData.observeForever(LiveData.java:207)
        at android.arch.lifecycle.MediatorLiveData$Source.plug(MediatorLiveData.java:141)
        at android.arch.lifecycle.MediatorLiveData.onActive(MediatorLiveData.java:118)
        at android.arch.lifecycle.LiveData$ObserverWrapper.activeStateChanged(LiveData.java:410)
        at android.arch.lifecycle.LiveData$LifecycleBoundObserver.onStateChanged(LiveData.java:368)
        at android.arch.lifecycle.LifecycleRegistry$ObserverWithState.dispatchEvent(LifecycleRegistry.java:354)
        at android.arch.lifecycle.LifecycleRegistry.forwardPass(LifecycleRegistry.java:292)
        at android.arch.lifecycle.LifecycleRegistry.sync(LifecycleRegistry.java:332)
        at android.arch.lifecycle.LifecycleRegistry.moveToState(LifecycleRegistry.java:137)
        at android.arch.lifecycle.LifecycleRegistry.handleLifecycleEvent(LifecycleRegistry.java:123)
        at android.support.v4.app.Fragment.performStart(Fragment.java:2380)
        at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1467)
        at android.support.v4.app.FragmentManagerImpl.performPendingDeferredStart(FragmentManager.java:1232)
        at android.support.v4.app.FragmentManagerImpl.startPendingDeferredFragments(FragmentManager.java:1855)
        at android.support.v4.app.FragmentManagerImpl.doPendingDeferredStart(FragmentManager.java:2697)
        at android.support.v4.app.FragmentManagerImpl.execSingleAction(FragmentManager.java:2221)
        at android.support.v4.app.BackStackRecord.commitNowAllowingStateLoss(BackStackRecord.java:649)
        at android.support.v4.app.FragmentPagerAdapter.finishUpdate(FragmentPagerAdapter.java:145)
        at android.support.v4.view.ViewPager.populate(ViewPager.java:1238)
        at android.support.v4.view.ViewPager.populate(ViewPager.java:1086)
        at android.support.v4.view.ViewPager$3.run(ViewPager.java:267)
        at android.support.v4.view.ViewPager.completeScroll(ViewPager.java:1999)
        at android.support.v4.view.ViewPager.onInterceptTouchEvent(ViewPager.java:2129)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2511)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2968)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2600)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2968)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2600)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2968)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2600)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2968)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2600)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2968)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2600)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2968)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2600)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2968)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2600)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2968)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2600)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2968)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2600)
        at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:448)
        at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1829)
        at android.app.Activity.dispatchTouchEvent(Activity.java:3307)
        at android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:68)
        at android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:68)
        at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:410)
2019-01-17 10:24:03.777 9049-9049/org.wordpress.android.beta E/AndroidRuntime:     at android.view.View.dispatchPointerEvent(View.java:12015)
        at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4795)
        at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4609)
        at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4147)
        at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4200)
        at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4166)
        at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4293)
        at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4174)
        at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:4350)
        at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4147)
        at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4200)
        at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4166)
        at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4174)
        at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4147)
        at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:6661)
        at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:6635)
        at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:6596)
        at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6764)
        at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:186)
        at android.view.InputEventReceiver.nativeConsumeBatchedInputEvents(Native Method)
        at android.view.InputEventReceiver.consumeBatchedInputEvents(InputEventReceiver.java:177)
        at android.view.ViewRootImpl.doConsumeBatchedInput(ViewRootImpl.java:6735)
        at android.view.ViewRootImpl$ConsumeBatchedInputRunnable.run(ViewRootImpl.java:6787)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:911)
        at android.view.Choreographer.doCallbacks(Choreographer.java:723)
        at android.view.Choreographer.doFrame(Choreographer.java:652)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:897)
        at android.os.Handler.handleCallback(Handler.java:790)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6494)
        	... 3 more
```

File: WordPress/src/main/res/layout/stats_block_empty_item.xml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
               android:layout_width="match_parent"
+             android:layoutDirection="rtl"
```
[the remark this answers is not in the document]
Comment (planarvoid) [reply WordPress/src/main/res/layout/stats_block_empty_item.xml]: nope, I'll remove it

**Comment by planarvoid** [comment]
> I think the rows in sections/blocks should be aligned right, wdyt?

I think this is caused by the english being selected. There is nothing in the layout that aligns the text left/right.

I'll look into the crash

**Comment by planarvoid** [comment]
@malinajirka this should be ready for another check

**Comment by malinajirka** [comment]
Thanks! LGTM

**Pull request merged**